tools-ui/src/main/java/com/hedera/hashgraph/client/ui/CreatePaneController.java

```diff
@@ -2136,17 +2134,14 @@ private void setLocalDateString(DatePicker datePicker, TextField hourField, Text
 			return;
 		}
 
-
 		var localDateTime = LocalDateTime.of(datePicker.getValue(),
 				LocalTime.of(Integer.parseInt(hourField.getText()), Integer.parseInt(minuteField.getText()),
 						Integer.parseInt(secondsField.getText())));
 		var transactionValidStart = Date.from(localDateTime.atZone(ZoneId.of(timeZone.getID())).toInstant());
 
-		if (transactionValidStart.toInstant().isBefore(Instant.now())) {
-			label.setStyle("-fx-text-fill: red");
-		} else {
-			label.setStyle("-fx-text-fill: black");
-		}
+		final var beforeNowBoolean = transactionValidStart.toInstant().isBefore(Instant.now());
+		label.setStyle("-fx-text-fill: " + (beforeNowBoolean ? "red" : "black"));
+		invalidDate.setVisible(beforeNowBoolean);
```

Comment:
nit: can drop the `Boolean` in the name.

Reply:
Thanks, I will change it in the next pr